docs/tutorial/tutorial-0.rst

```diff
@@ -11,7 +11,8 @@ button, that prints to the console when you press the button.
 Set up your development environment
 ===================================
 
-Open a command prompt on your computer and make sure that you can successfully run the :code:`python3` command. Create a working directory for your code and change to it.
+Make sure you installed `Toga prerequisites <https://github.com/beeware/toga#prerequisites>`_, such as Python3 and other librariesn the create a working directory for your code and change to it.
```

Comment:
> Make sure you installed `Toga prerequisites <https://github.com/beeware/toga#prerequisites>`_, such as Python3 and other librariesn the create a working directory for your code and change to it.

Make sure you installed the `Toga prerequisites <https://github.com/beeware/toga#prerequisites>`_, such as Python 3 and the other libraries. Then create a working directory for your code and change to it.

Reply:
Oops!